libs/hpc-data/src/lib/flowObject.ts

```diff
@@ -0,0 +1,22 @@
+import * as t from 'io-ts';
+
+const FLOW_REF_DIRECTION = t.keyof({
+  source: null,
+  destination: null,
+});
```

Comment:
If you are trying to define the possible values of `refDirection` I think what you are trying to do here is:
```suggestion
const FLOW_REF_DIRECTION = t.union([t.literal('source'),t.literal('destination')]);
```